src/main.rs

```diff
@@ -28,6 +32,68 @@ fn main() -> Result<(), JaktError> {
     }
 }
 
+/// Make sure to keep these up-to-date if you're adding arguments.
+const USAGE: &str = "usage: jakt [-h] [FILES...]";
+
+// FIXME: Once format is stable as a const function, include USAGE in this string.
+const HELP: &str = "\
+Flags:
+  -h, --help           Print this help and exit 
+
+Options:
+  FILES...             List of files to compile. The output is `output.cpp`
+                       in the cwd.
+
+Jakt programming language (c) 2022 the SerenityOS developers.
+Licensed under BSD 2-Clause, see LICENSE.
```

Comment:
I don't think this needs to appear in CLI output

Reply:
Maybe I've seen this too often in help output